h2o-core/src/main/java/water/ExternalFrameHandler.java

```diff
@@ -70,7 +71,7 @@
     /**
      * Method which receives the {@link SocketChannel} and {@link AutoBuffer} and dispatches the request for further processing
```

Comment:
Basic tests for original functionality exist and are here https://github.com/h2oai/h2o-3/blob/master/h2o-core/src/test/java/water/ExternalFrameReaderClientTest.java and here https://github.com/h2oai/h2o-3/blob/master/h2o-core/src/test/java/water/ExternalFrameWriterClientTest.java